src/analysisd/decoders/winevtchannel.c

```diff
@@ -113,6 +113,7 @@ int DecodeWinevt(Eventinfo *lf){
     os_calloc(OS_MAXSTR, sizeof(char), join_data);
 
     find_event = strstr(lf->log, "Event");
+    mdebug1("Received message: %s", lf->log);
```

Comment:
This message can be very noisy since it will print the full events received from all the Windows agents. Please remove it.